tests/html_repr/html_via_hypothesis_test.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Should we place hypothesis test in a separate folder (subfolder of `tests/`?) so it is easier to run them independently?

Reply:
I thought that would make it easier to exclude/ignore them, using, e.g., `--ignore` or something in `setup.cfg`?

Reply:
Moved it into `tests/hypothesis`. The tests are run by default but can be disabled using `--ignore=tests/hypothesis`.